var/spack/repos/builtin/packages/py-tensorboard-data-server/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+
+from spack import *
+
+
+class PyTensorboardDataServer(PythonPackage):
+    """Fast data loading for TensorBoard"""
+
+    homepage = "https://github.com/tensorflow/tensorboard/tree/master/tensorboard/data/server"
+    git      = "https://github.com/tensorflow/tensorboard"
+
+    version('0.6.1', commit='6acf0be88b5727e546dd64a8b9b12d790601d561')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-pip', type='build')
+    depends_on('rust', type='build')
+
+    phases = ['build', 'install']
```

Comment:
Instead of overriding phases I would just use `@run_before('install')` to decorate `build`. That way we don't lose things like post-install tests.

Reply:
I think phases makes more sense. How would using phases break post-install tests?

Reply:
The post-install tests are registered to the phase when the base class is initialized. If you override `phases`, all tests are deregistered and you have to add them again. @alalazo is this understanding correct?